### PR TITLE
03 1453: Make a reusable form for creating and editing an appointment.

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
- Instead of having two separate duplicate forms for adding and editing an appointment, make one that is reusable.
- Also fix the `Undefined undefined` patient name in the banner in the appointments form.
- Also fixed some layout bugs in the form

## Screenshots
- Dev3 recording with broken banner and layout

https://user-images.githubusercontent.com/30952856/189104031-8a75e1db-7e7f-43ac-b8e9-bb2d4b74ea27.mov

- Fixed in this PR

https://user-images.githubusercontent.com/30952856/189105410-d377b527-f62f-4f41-8126-9438095504a2.mov

## Related Issue

[Make a reusable form for creating and editing an appointment.](https://issues.openmrs.org/projects/O3/issues/O3-1453)

### CODE REVIEW

File: packages/esm-appointments-app/src/appointment-forms/appointments-form.component.tsx [hunk 1]
```diff
@@ -138,8 +165,8 @@ const AppointmentForm: React.FC<AppointmentFormProps> = ({ appointment }) => {
           datePickerType="single"
           id="visitDate"
           light
-          style={{ paddingBottom: '1rem' }}
-          minDate={new Date().toISOString()}
+          style={{ paddingBottom: '1rem', marginLeft: '12.5rem' }}
```
Comment (CynthiaKamau) [inline packages/esm-appointments-app/src/appointment-forms/appointments-form.component.tsx:168]: Kindly move the styles to the scss file

**Review by CynthiaKamau — COMMENTED** [review]
Great job @hadijahkyampeire , i have left a comment

File: packages/esm-appointments-app/translations/en.json [hunk 2]
```diff
@@ -3,17 +3,13 @@
...
   "anotherVisitType": "Start another visit type",
   "appointmentDateAndTime": "Appointments Date and Time",
-  "appointmentDateTime": "Appointment Date and Time",
   "appointmentForm": "Appointments Form",
   "appointmentFormError": "Error scheduling appointment",
   "appointmentHistory": "Appointment History",
   "appointmentList": "Appointments list",
   "appointmentMetrics": "Appointment metrics",
-  "appointmentNoteLabel": "Write an additional note",
-  "appointmentNotePlaceholder": "Write any additional points here",
```
Comment (ZacButko) [inline packages/esm-appointments-app/translations/en.json:13]: Some of these labels and placeholders were more verbose before. Do we really want to lose this verbage?

**Review by ZacButko — APPROVED** [review]
ok

File: packages/esm-appointments-app/translations/en.json (hunk 2 above)
Comment (hadijahkyampeire) [reply packages/esm-appointments-app/translations/en.json:13]: I will maintain the wordings then.

**Pull request merged**